src/pages/en/guides/markdown-content.md

```diff
@@ -301,7 +301,7 @@ const posts = await Astro.glob('./posts/**/*.md');
 
 #### `Content`
 
-A component that renders the contents of the Markdown file. Here is an example:
+A component that returns the full rendered contents of the Markdown file. Here is an example:
```

Comment:
Nit, but would it be better to stick to “renders” rather than “returns” here? Like, you don’t do `const contents = Component()` so I’m not sure “returning” is quite the right concept here (even if that is what technically happens under the hood).

```suggestion
A component that renders the full contents of the Markdown file. Here is an example:
```

Reply:
"To return or not to return, it really depends on how you throw...." - Fuzzy